routes/api.php

```diff
@@ -42,3 +42,11 @@
      */
     Route::apiResource('books', 'BookController')->names('books');
 });
+
+// Authentication with sessions, specifically for AJAX from Blade.
+Route::group(['middleware' => ['auth', 'web'], 'as' => 'ajax.', 'namespace' => 'Ajax'], function () {
```

Comment:
Menyesuaikan yang di atas.

```suggestion
Route::group(['middleware' => ['auth', 'web'], 'as' => 'api.'], function () {
```